### PR TITLE
[SPARK-35001][PYTHON] Use ps as the short name instead of pp in pandas-on-Spark

### PR DESCRIPTION
**Pull request by HyukjinKwon**
### What changes were proposed in this pull request?

This PR proposes to fix:

```python
import pyspark.pandas as pp
```

to

```python
import pyspark.pandas as ps
```

### Why are the changes needed?

`pp` might sound offensive in some contexts.

### Does this PR introduce _any_ user-facing change?

The change is in master only. We'll use `ps` as the short name instead of `pp`.

### How was this patch tested?

The CI in this PR will test it out.

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
cc @ueshin, @itholic, and @xinrong-databricks

File: python/pyspark/pandas/accessors.py [hunk 1]
```diff
@@ -265,24 +265,24 @@ def apply_batch(self, func, args=(), **kwds) -> "DataFrame":
         DataFrame.applymap: For elementwise operations.
         DataFrame.aggregate: Only perform aggregating type operations.
         DataFrame.transform: Only perform transforming type operations.
-        Series.koalas.transform_batch: transform the search as each pandas chunpp.
+        Series.koalas.transform_batch: transform the search as each pandas chunks.
```
Comment (itholic) [inline python/pyspark/pandas/accessors.py:268]: Thanks for fixing this  😅

**Review by itholic — APPROVED** [review]
LGTM if tests pass

btw, I created related JIRA ticket at SPARK-34983 before.

Seems like we should close both SPARK-35001 and SPARK-34983.

**Comment by xinrong-databricks** [comment]
Thanks! Is this PR aimed to change the main code only?

**Comment by HyukjinKwon** [comment]
I am closing in favour of https://github.com/apache/spark/pull/32108

**Pull request closed without merging**